Sources/KituraContracts/CodableQuery/QueryEncoder.swift

```diff
@@ -152,6 +161,9 @@ public class QueryEncoder: Coder, Encoder, BodyEncoder {
      */
     public func encode<T: Encodable>(_ value: T) throws -> [String : String] {
         let encoder = QueryEncoder()
```

Comment:
If the user had customized `QueryEncoder.dateEncoder` then it would be lost here.  I think you need
```swift
encoder.dateEncoder = self.dateEncoder
```